skins/laika/src/components/pages/donation_form/AddressType.vue

```diff
@@ -41,13 +42,29 @@ export default Vue.extend( {
 			type: AddressTypeModel.PERSON,
 		};
 	},
+	props: {
+		disabledAddressTypes: Array,
+	},
 	computed: {
 		AddressTypeModel: {
 			get: function () {
 				return AddressTypeModel;
 			},
 		},
 	},
+	watch: {
+		disabledAddressTypes:
+		{
+			handler: function ( disabledAddressTypes ) {
+				const $this = ( this as any );
```

Comment:
I don't get why TypeScript randomly starts losing its mind with this just because I refactored this function slightly. This was my workaround to get it to work but no clue if it's any good @gbirke.